go.sum

```diff
@@ -436,8 +436,8 @@ github.com/yuin/goldmark v1.1.32/go.mod h1:3hX8gzYuyVAZsxl0MRgGTJEmQBFcNTphYh9de
 github.com/yuin/goldmark v1.2.1/go.mod h1:3hX8gzYuyVAZsxl0MRgGTJEmQBFcNTphYh9decYSb74=
 github.com/yuin/goldmark v1.3.5/go.mod h1:mwnBkeHKe2W/ZEtQ+71ViKU8L12m81fl3OWwC1Zlc8k=
 github.com/yuin/goldmark v1.4.13/go.mod h1:6yULJ656Px+3vBD8DxQVa3kxgyrAnzto9xy5taEt/CY=
-go.mongodb.org/atlas v0.25.1-0.20230522162956-d4f376b72f78 h1:JafEV07Uj8FCsIpf3q2I3l64rlS9KrVdIFarlL9LZ8s=
-go.mongodb.org/atlas v0.25.1-0.20230522162956-d4f376b72f78/go.mod h1:sZ7wAszLXP2JNaWysQnvnL7cB2d32noqUi3vr2IPF5s=
+go.mongodb.org/atlas v0.26.2-0.20230529122338-7389eac5a4dd h1:yXAJNFz8e93a1OzxPZ3jYXzne99N1wFnewuuO15IM/I=
```

Comment:
can you do a release of the SDK and ping here now to the stable branch to get back dependabot?